noxfile.py

```diff
@@ -40,24 +48,127 @@ def install_core(session: nox.Session) -> None:
     )
 
 
-@nox.session(python=False, name="install-client")
-def install_client(session: nox.Session) -> None:
-    session.cd("client")
+@nox.session(python=False, name="build-go")
+def build_go(session: nox.Session) -> None:
```

Comment:
This code is very similar to core/setup.py; is that file meant to be removed?

Reply:
yep exactly. the idea is to split the build part of the go code and the packaging into python into 2 different "actions/targets".